registry/donor/views.py

```diff
@@ -106,6 +106,20 @@ def detail(rc):
     )
 
 
+@blueprint.get("/detail/<rc>/award_document/<medal_slug>/")
+@login_required
+def render_award_document(rc, medal_slug):
+    overview = DonorsOverview.query.get_or_404(rc)
+    medal = Medals.query.filter_by(slug=medal_slug).first_or_404()
+
+    return render_template(
+        "donor/award_document.html",
+        overview=overview,
+        medal=medal,
+        today=datetime.now().strftime("%d.%m.%Y"),
```

Comment:
Mezi číslovkami by měla být mezera a pomlčka ve formátu nás zbaví nul na začátku jednociferných čísel.
```suggestion
        today=datetime.now().strftime("%-d. %-m. %Y"),
```